drivers/hmis/spec/models/hmis/form/form_processor_spec.rb

```diff
@@ -516,14 +516,26 @@
 
   describe 'Form processing for Clients' do
     let(:definition) { Hmis::Form::Definition.find_by(role: :CLIENT) }
+    let(:primary_name) do
+      {
+        primary: true,
+        first: 'Terry',
+        middle: 'Mid',
+        last: 'Breeze',
+        suffix: 'Jr',
+        nameDataQuality: 'FULL_NAME_REPORTED',
+      }
+    end
+    let(:secondary_name) do
+      {
+        primary: false,
+        first: 'Gerome',
+        nameDataQuality: 'PARTIAL_STREET_NAME_OR_CODE_NAME_REPORTED',
```

Comment:
Did you mean to include `STREET` above?  Maybe that should be `STREET_NAME_OR_PARTIAL_NAME_OR_CODE_NAME_REPORTED`?  Weird that it's a partial street name, like Broadway Avenue being reported as Broad Ave.

Reply:
These are code-generated from the HUD Enum descriptions. For this one, it is `Partial, street name, or code name reported`. I should probably be better about looking up the enum by value (2) instead of writing it directly in a constant, since we'll have to change them when descriptions change.